src/decorations/intervals.jl

```diff
@@ -92,9 +98,9 @@ macro decorated(ex...)
     local x
 
     if length(ex) == 1
-        x = :(@interval($(esc(ex[1]))))
+        x = :($(esc(ex[1])))
```

Comment:
You may be right, though tests pass. I'll revert that change, because it is actually not needed. Thanks!